### PR TITLE
commit message

### PR DESCRIPTION
**Pull request by OlenaLifyrenko**
:)

### CODE REVIEW

File: some-doc.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+i am studying GIT
+1
```
Comment (alifyrenko) [inline some-doc.txt]: should be '2'

File: some-doc.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+i am studying GIT
```
Comment (alifyrenko) [inline some-doc.txt]: i - with capital letter

Comment (OlenaLifyrenko) [reply some-doc.txt]: fixed

File: some-doc.txt (hunk 1 above)
Comment (OlenaLifyrenko) [reply some-doc.txt]: fixed

**Review by alifyrenko — APPROVED** [review]

**Pull request merged**